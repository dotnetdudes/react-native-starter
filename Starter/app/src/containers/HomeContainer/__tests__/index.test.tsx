import React from 'react';
import { HomeContainer } from '../index';
import { render, screen } from '@testing-library/react-native';
// import { screen } from '@testing-library/react-native';
// import { customRender } from '@/utils/test-utils';
import renderer from 'react-test-renderer';

describe('Testing react navigation', () => {
  test('clicking on the button takes you to the notifications screen', () => {
    const component = <HomeContainer />;

    render(component);
    const oldScreen = screen.queryByText('hi there');

    expect(oldScreen).toBeOnTheScreen();
  });
});

test('renders correctly', () => {
  const tree = renderer.create(<HomeContainer />).toJSON();
  expect(tree).toMatchSnapshot();
});
