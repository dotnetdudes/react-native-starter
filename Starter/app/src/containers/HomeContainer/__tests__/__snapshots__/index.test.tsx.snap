// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTScrollView>
  <View>
    <View>
      <Text>
        hi there
      </Text>
    </View>
  </View>
</RCTScrollView>
`;
